setup.py

```diff
@@ -56,7 +56,7 @@
         "docutils==0.20.*",
         "libcst==1.*",
         "platformdirs==4.*",
-        "sphinx==7.*",
+        "sphinx>=7",
```

Comment:
@LilSpazJoekp something like "sphinx>=7,<9" would be more conservative: if in the future sphinx 9 breaks backward compatibility, it won't be installed by default and it won't break builds.
If sphinx 9 will not break backward compatibility, the range can be changed easily.